atmat/lattice/atsimplering.m

```diff
@@ -115,9 +128,9 @@
 nonlin_elem=atbaselem('NonLinear','DeltaQPass','Class','NonLinear',...
     'Betax',betax,'Betay',betay,...
     'Alphax',alphax,'Alphay',alphay,...
-    'Qpx',qpx,'Qpy',qpy,...
+    'chromx_arr',chromx_arr,'chromy_arr',chromy_arr,...
```

Comment:
You should add also the field chrom_maxorder in the nonlinear element.

Reply:
done